<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diagram</class>
 <widget class="QDialog" name="Diagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  none;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
   </property>
   <property name="title">
    <string>Zależność temperatury od czasu</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="styleSheet">
       <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>491</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
   </property>
   <property name="title">
    <string>Zależność natężenia światła od czasu</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="plot_light" native="true">
      <property name="styleSheet">
       <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>280</width>
     <height>107</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="bx_x"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

border-radius: 4px;
  background-color: rgb(5, 6, 45);
  border: none;
  color: #FFFFFF;
  text-align: center;
  padding: 5px;

  white-space: nowrap;

}

.QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: rgb(255, 255, 255);
}

.QPushButton:pressed {
    background-color: rgb(5, 6, 45);
    border: 2px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
}</string>
         </property>
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="bx_y"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelY">
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: none;</string>
         </property>
         <property name="text">
          <string>   Y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonClear">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

border-radius: 4px;
  background-color: rgb(5, 6, 45);
  border: none;
  color: #FFFFFF;
  text-align: center;
  padding: 5px;

  white-space: nowrap;

}

.QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: rgb(255, 255, 255);
}

.QPushButton:pressed {
    background-color: rgb(5, 6, 45);
    border: 2px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
}</string>
         </property>
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelX">
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: none;</string>
         </property>
         <property name="text">
          <string> X</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>280</width>
     <height>107</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">
background-color: rgba(255, 255, 255, 0.5);
  border: 2px solid qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: black;
  font-family: Phantomsans, sans-serif;
  text-decoration: none;</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonAddLight">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

border-radius: 4px;
  background-color: rgb(5, 6, 45);
  border: none;
  color: #FFFFFF;
  text-align: center;
  padding: 5px;

  white-space: nowrap;

}

.QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: rgb(255, 255, 255);
}

.QPushButton:pressed {
    background-color: rgb(5, 6, 45);
    border: 2px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
}</string>
         </property>
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelXLight">
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: none;</string>
         </property>
         <property name="text">
          <string> X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonClearLight">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

border-radius: 4px;
  background-color: rgb(5, 6, 45);
  border: none;
  color: #FFFFFF;
  text-align: center;
  padding: 5px;

  white-space: nowrap;

}

.QPushButton:hover {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
  color: rgb(255, 255, 255);
}

.QPushButton:pressed {
    background-color: rgb(5, 6, 45);
    border: 2px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(175, 64, 255, 255), stop:0.5 rgba(91, 66, 243, 255), stop:1 rgba(0, 221, 235, 255));
}</string>
         </property>
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="bx_yLight"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="bx_xLight"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelYLight">
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: none;</string>
         </property>
         <property name="text">
          <string>   Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
